### DISCUSSION TITLE
Color customization

### DISCUSSION THREAD

**Discussion by CommonClimate** [alshedivat/al-folio#399] [Q&A] [answered]
Hi, I am new to jekyll and to this theme, so please forgive my naive question. 

How do I change the theme's colors?  The instructions say "you can quickly change it by editing `$theme-color` variable in the `_sass/_themes.scss` file. Other color variables are listed there as well." 

Except I don't see a list of possible colors there, except the ones already used by default (purple, black, gray). Is there a way to provide custom HTML color codes, for instance?

I could not find a post in this discussion to address my issue, but please correct me if this is an oversight.  

Thanks for any pointers.
Best,
Julien

**Comment by rohandebsarkar** [comment, marked as answer]
All the default colors are initialized in `_sass/_variables.scss` file. You can add you custom colors to that file and modify the theme according to your needs.

**Reply by CommonClimate** [reply]
Thank you! Not speaking SASS, I wasn't sure where all that was hidden, so I am grateful for the pointer.
